app/controllers/lessons_controller.rb

```diff
@@ -43,5 +43,7 @@ def lesson_params
 
   def lesson_find
     @lesson = Lesson.all.find(params[:id])
+  rescue StandardError => e
```

Comment:
This is not correct. We should return 404 error only in case of missing record. In other cases we need to return 400 error code.

Reply:
We can remove this rescue block at all, and rewrite our error pages to the JSON.

Reply:
It will be nice to investigate question about errors handling in a separate ticket.